roxie/ccd/ccdqueue.cpp

```diff
@@ -490,6 +482,49 @@ class CRoxieQueryPacket : implements IRoxieQueryPacket, public CInterface
                 finger++;
             }
             traceLength = finger - traceInfo;
+        }
+    }
+
+    ~CRoxieQueryPacketBase()
+    {
+        free(data);
+    }
+
+};
+
+// MORE - this is for TESTING ONLY - do not release with this key here like this!
```

Comment:
Does 'release' exclude 'merge into master branch'?

Reply:
There is another Jira open to fix it properly, but this change allows us to do some performance testing of this option in master. Merging it to master is acceptable so long as that other Jira gets merged before 8.0